part4/performance-tuning.md

```diff
@@ -28,6 +28,12 @@ If you have generated the tables automatically using your JPA implementation (e.
 -   Sagas
 
     Table 'AssociationValueEntry', columns `associationKey` and `sagaId`,
+    
+    Table 'AssociationValueEntry', columns `sagaId`,
+    
+    Table 'AssociationValueEntry', columns `sagaType` and `sagaId`,
```

Comment:
I believe, by turning these two columns around, that the index on 'sagaId' alone should not be of any added value. Furthermore, it is best to have the most unique column in front of any columns that contains less unique values. Hence, 'sagaId', 'sagaType' would be preferred.

Also, we'd probably need an index on 'associationKey' and 'sagaType' for the "findSagas" functionality.